gradle/dependency-versions.gradle

```diff
@@ -25,10 +25,10 @@
   junitVersion = "4.8.1"
   mockitoVersion = "1.8.4"
   scalaTestVersion = "2.2.4"
-  zkClientVersion = "0.3"
+  zkClientVersion = "0.9"
```

Comment:
Just curious. Kafka 0.10.0.1 version depends upon zkClient version 0.8. https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.11/0.10.0.1. Any reason for using zkClient with version 0.9 ?